src/Protocol/ActivityPub/Receiver.php

```diff
@@ -1890,18 +1890,15 @@ private static function processObject(array $object, string $actor)
 		$object_data['receiver_urls']  = self::getReceiverURL($object);
 		$object_data['receiver']       = $receivers;
 		$object_data['reception_type'] = $reception_types;
+		$object_data['unlisted'] = false;
```

Comment:
Please align the assignment operators.
```suggestion
		$object_data['unlisted']       = false;
```